docs/tutorials/otio-filebundles.md

```diff
@@ -2,91 +2,122 @@
 
-## MediaReferencePolicy
+Extract the contents of the bundle and convert to an rv playlist:
 
-When building a file bundle using the OTIOZ/OTIOD adapters, you can set the 'media reference policy', which is described by an enum in the file_bundle_utils module.  The policies can be:
+`otioconvert -i /var/tmp/somefile.otioz -a extract_to_directory=/var/tmp/somefile -o /var/tmp/somefile/somefile.rv`
 
-- (default) ErrorIfNotFile: will raise an exception if a media reference is found that is of type `ExternalReference` but that does not point at a `target_url`.
-- MissingIfNotFile: will replace any media references that meet the above condition with a `MissingReference`, preserving the original media reference in the metadata of the new `MissingReference`.
-- AllMissing: will replace all media references with `MissingReference`, preserving the original media reference in metadata on the new object.
+## Write Adapter
 
-When running in `AllMissing` mode, no media will be put into the bundle.
+### Source Timeline Constraints
+
+For creating otio bundles using the provided python adapter, an OTIO file is used as input.  There are some constraints on the source timeline.
 
-## OTIOD
+#### Unique Basenames
 
-The OTIOD adapter will build a bundle in a directory stucture on disk.  The adapter will gather up all the files it can and copy them to the destination directory, and then build the `.otio` file with local relative path references into that directory.
+Because file bundles have a flat namespace for media, and media will be copied into the bundle, the `ExternalReference` media references in the source OTIO must have a target_url fields pointing at media files with unique basenames.  For example, if there are media references that point at:
 
-## OTIOZ
+`/project_a/academy_leader.mov`
+
+and:
 
-The OTIOZ adapter will build a bundle into a zipfile (using the zipfile library).  The adapter will write media into the zip file uncompressed and the content.otio with compression.
+`/project_b/academy_leader.mov`
 
-### Optional Arguments:
+Because the basename of both files is `academy_leader.mov`, this will be an error.  The adapters have different policies for how to handle media references.  See below for more information.
 
-- Read:
-    - extract_to_directory: if a value other than `None` is passed in, will extract the contents of the bundle into the directory at the path passed into the `extract_to_directory` argument.
+#### Expected Source Timeline External Reference URL Format
 
-## Example usage in otioconvert
+The file bundle adapters expect the `target_url` field of any `media_reference`s in the source timeline to be in one of two forms (as produced by python's urlparse library):
 
-### Convert an otio into a zip bundle
+- absolute path:  "file:///path/to/some/file" (encodes "/path/to/some/file")
+- relative path: "path/to/some/file" (assumes the path is relative to the current working directory when invoking the adapter on the source timeline).
```

Comment:
an update on this - it would appear that it refers to the cwd of the command being run, but I think it should probably be relative to the location of the timeline.  The final resulting paths are relative to the root of the bundle.

Reply:
After going through the code, I think the right thing to do is separate the change into a separate PR, because that changes behavior.  This PR only changes documentation.  Does that make sense, to split that off?  I *think* its a pretty straightforward change, here:

https://github.com/AcademySoftwareFoundation/OpenTimelineIO/blob/5dff8be58d1c4b63040fc70c8afd261bd31d6e55/src/py-opentimelineio/opentimelineio/adapters/file_bundle_utils.py#L132

That would need take the path to the input otio file (or maybe more directly a CWD that the calling functions pass as the path to the input otio file) and use that to construct the absolute path (rather than relying on `os.path.abspath()` to do it).

Reply:
I created a separate issue to track that.  I think it might need some discussion:  https://github.com/AcademySoftwareFoundation/OpenTimelineIO/issues/1783